Mantis.podspec

```diff
@@ -21,7 +21,7 @@ Pod::Spec.new do |s|
   s.social_media_url   = "http://twitter.com/guoyingtao"
   s.platform     = :ios
   s.swift_version = "5.0"
-  s.ios.deployment_target = "11.0"
```

Comment:
Is this change really necessary? I still have `deployment_target = "12.0"` for my app

Reply:
It is because of my previous PR #169 
I added a CropToolbarIconProvider also I decided to use SF Symbol for CropToolbar icons instead of drawing them by myself which needs iOS 13
With apple's iOS 16 coming soon, I think it might be a good timing to move to iOS 13 because it also supports Combine, SwiftUI etc.